src/Symfony/Component/Form/Tests/AbstractRequestHandlerTest.php

```diff
@@ -280,32 +278,10 @@ public function testSubmitMultipleFiles($method)
             'param3' => $this->getMockFile('3'),
         ]);
 
-        $form->expects($this->once())
-             ->method('submit')
-             ->with($file, 'PATCH' !== $method);
-
         $this->requestHandler->handleRequest($form, $this->request);
-    }
-
-    /**
-     * @dataProvider methodExceptGetProvider
-     */
-    public function testSubmitFileWithNamelessForm($method)
```

Comment:
This test was completely broken and just worked because our mocked form behaved completely different from how a real form would behave. I am looking into a proper fix in a different PR.